### DISCUSSION TITLE
Cant verify philippine number

### DISCUSSION THREAD

**Discussion by Vince-vegas** [vercel/vercel#5342] [Help] [answered]
Hi Vercel Team,

Why I cant sign-up into your platform using the philippine number. When I Sign-up it ask me for a number then I choose the Philippine number after that the error appeared **Invalid phone number**

**Comment by runtechspace** [comment]
I'm have the same issue but with a Nigerian number

**Comment by swarnava** [comment, marked as answer]
Please reach out to registration@vercel.com using your registered email so that our team can look into this!

**Comment by runtechspace** [comment]
Okay, thanks
